web-ui/src/main/resources/catalog/components/viewer/projectionSwitcher/ProjectionSwitcher.js

```diff
@@ -69,6 +69,8 @@
                                   this.changeLayerProjection(subLayer, oldProj,
                                       newProj);
                                 });
+                          } else if (layer instanceof ol.layer.Image) {
+                            gnMap.updateWmsLayerForProj(scope.map, layer, newProj);
```

Comment:
Also update and show WMS layers whenever the projection changes (they were ignored until now).